### PR TITLE
Add zipkin.proto

### PR DESCRIPTION
**Pull request by jan25**
Signed-off-by: Abhilash Gnan <abhilashgnan@gmail.com>

- Adding `zipkin.proto` file to support sending Zipkin Protobuf spans to jaeger-collector
- Parent issue: https://github.com/jaegertracing/jaeger/pull/1695

### CODE REVIEW

File: proto/zipkin.proto [hunk 1]
```diff
@@ -0,0 +1,233 @@
+//
```
Comment (yurishkuro) [inline proto/zipkin.proto]: please indicate the origin (url) of this file

Comment (jan25) [reply proto/zipkin.proto]: Done. We didn't add URL for [swagger](https://github.com/jaegertracing/jaeger-idl/blob/master/swagger/zipkin2-api.yaml) and [thrift](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/zipkincore.thrift), should we add for them too?

File: proto/zipkin.proto [hunk 2]
```diff
@@ -0,0 +1,233 @@
+// https://github.com/openzipkin/zipkin-api/blob/master/zipkin.proto
```
Comment (yurishkuro) [inline proto/zipkin.proto]: nit: Copied from ... (followed by blank line)

**Comment by jan25** [comment]
Looks like zipkin swagger and thrift model files have been updated on upstream and we have pretty old ones. We can update them in a separate PR.

**Comment by yurishkuro** [comment]
Thanks. We should also add a build step as a validation of .proto files, similar to how we build from thrift.

**Pull request merged**